src/renderers/WebGLRenderer.js

```diff
@@ -3464,6 +3025,26 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.allocTextureUnit = allocTextureUnit;
+	this.setTexture2D = setTexture2D;
```

Comment:
@mrdoob 
> Should be this.setTexture?

I renamed it for clarity and didn't want to put the full detection in there for performance reasons. It's being called both from `WebGLUniforms` and from an inner loop of the sprite plugin - both already know what they want.

> Or, at least we should add `this.setTexture = function ( texture, slot ) { console.warn(); return setTexture2D( texture, slot ); }`

Agreed. 

Or don't warn and just implement it in the obvious way, auto-detecting what kind of texture is being passed? Although, that is probably not all that useful.